docs/installation/installation.md

```diff
@@ -10,8 +10,9 @@
   - [Watching Namespaces](#watching-namespaces)
   - [Watching custom ingress class](#watching-custom-ingress-class)
   - [Installing cert-manager](#installing-cert-manager)
-- [Installing CRDs](#installing-crds)
-- [Installing Tyk Operator](#installing-tyk-operator)
+- [Installing Tyk Operator and CRDs](#installing-tyk-operator-and-crds)
+  - [Using Helm Chart](#helm-chart)
+  - [Tyk Operator Repository](#tyk-operator-repository)
 - [Upgrading Tyk Operator](#upgrading-tyk-operator)
```

Comment:
Add subheading `Using Helm Chart` and `Tyk Operator Repository` here too

Reply:
Added 👍